src/devtools/client/debugger/src/components/Editor/LineNumberTooltip.tsx

```diff
@@ -158,6 +159,16 @@ export default function LineNumberTooltip({
     }
   }, [analysisPointsCount]);
 
+  if (
+    breakpoints
+      .filter(b => !b.disabled)
+      .some(
+        b => b.location.sourceId === source?.id && b.location.line === lastHoveredLineNumber.current
+      )
```

Comment:
We could avoid the intermediate array creation and potentially over looping by
```suggestion
  breakpoints.some(
    b =>
      !b.disabled &&
      b.location.sourceId === source?.id &&
      b.location.line === lastHoveredLineNumber.current
  )
```